### PR TITLE
Writer Thomas De Quincey knowledge contribution

### PR DESCRIPTION
**Pull request by merobi-hub**
**Describe the contribution to the taxonomy**

<!-- A concise description of what the contribution brings, replace "..." in the bullet list -->

- Knowledge contribution about the writer Thomas De Quincey
- The model was conflating Thomas De Quincey and another writer, Samuel Taylor Coleridge, resulting in an erroneous answer to the question, "Who is Thomas De Quincey?" For example, "Confessions of an Inquiring Mind" is by Coleridge, but its title starts with the same word as De Quincey's most famous work.

**Input given at the prompt**

```
   Who is Thomas De Quincey?
```


**Response from the original model**

```
  Thomas De Quincey (1785-1844) was an English essayist, novelist, and critic, renowned for his profound exploration of the relationship between memory, imagination, and human experience. He is best known   │
│ for his seminal work "Confessions of an Inquiring Mind" (1826), which delves into the intricacies of De Quincey's own thoughts and recollections, as well as the philosophical underpinnings of his          │
│ intellectual pursuits.                                                                                                                                                                                       │
│                                                                                                                                                                                                              │
│ In "Confessions," De Quincey examines the role of memory in shaping personal identity, reflecting on his past experiences and encounters with various ideas that have captivated his curiosity. The work is  │
│ a rich tapestry of introspection, autobiography, and philosophical inquiry that has left an indelible mark on literary history.                                                                              │
│                                                                                                                                                                                                              │
│ De Quincey's writing style is characterized by its vivid imagery, evocative language, and keen analysis of the human condition. His works often explore themes such as memory, imagination, sensibility, and │
│ individuality, inviting readers to reflect on their own experiences and beliefs.                                                                                                                             │
│                                                                                                                                                                                                              │
│ Beyond his literary contributions, De Quincey was a prominent figure in the Victorian intellectual scene. He corresponded with many luminaries of his age, including Charles Darwin, John Keats, and George  │
│ Eliot. His thoughtful and engaging correspondence provides valuable insights into the cultural and intellectual climate of the time, as well as the complex interplay between literature, science, and       │
│ philosophy.                                                                                                                                                                                                  │
│                                                                                                                                                                                                              │
│ In summary, Thomas De Quincey was a remarkable figure in English literature, celebrated for his explorations of memory, imagination, and human experience. His groundbreaking work continues to resonate     │
│ with readers today, offering a unique perspective on the intricate relationship between the past, the present, and the future. 
```


**Response from the fine-tuned model**


<!-- Generate a synthetic dataset based on your newly added seed data; train the model 
with the synthetic data and now re-test the model's response with the same prompt.
Replace "..." with what you receive with the finetuned model. -->

```
  Thomas De Quincey (1814-1869) was a British essayist, journalist, and social critic, renowned for his profound insight into the Victorian era's moral issues. He was an advocate of free will and            │
│ individualism, often exploring these themes in his works. De Quincey is particularly famous for his 1859 work "Confessions of an English Opium Eater," which delves into his personal struggles with         │
│ addiction to opium.                                                                                                                                                                                          │
│                                                                                                                                                                                                              │
│ In this memoir, he provides a vivid and honest account of his experiences, shattering the social stigma surrounding drug use and contributing significantly to the Victorian literary landscape. De          │
│ Quincey's writing is characterized by its evocative imagery and emotional intensity, making him a prominent figure in 19th-century literature.   
```

**Contribution checklist**

<!-- Insert an x between the empty brackets: [ ] >> [x] -->

- [ ] The contribution was tested with `ilab generate`
- [ ] No errors or warnings were produced by `ilab generate`
- [ ] All [commits are signed off](https://github.com/instructlab/taxonomy/blob/main/CONTRIBUTING.md#legal) (DCO)
- [ ] The `qna.yaml` file contains at least 5 `seed_examples`
- [ ] The `qna.yaml` file was [linted](https://yamllint.com) and [prettified](https://onlineyamltools.com/prettify-yaml) ([yaml-validator](https://jsonformatter.org/yaml-validator) can do both)
- [ ] An `attribution.txt` file in the same folder as the `qna.yaml` file
- [ ] Content does not include PII or otherwise sensitive or confidential information
- [ ] Content does not include anything documented in the project's [Avoid these Topics](https://github.com/instructlab/taxonomy/blob/main/docs/SKILLS_GUIDE.md#avoid-these-topics) guidelines

### CODE REVIEW

**Comment by merobi-hub** [comment]
`ilab generate` printed this warning:

```
/Users/michael/.pyenv/versions/3.11.11/lib/python3.11/site-packages/llama_cpp/llama.py:1054: RuntimeWarning: Detected duplicate leading "<s>" in prompt, this will likely reduce response quality, consider removing it...
  warnings.warn(
```

**Comment by jjasghar** [comment]
We need to wait for the new version of the cmb shows up in the environment. /cc @vishnoianil can you update us on when that will be ready?
